### PR TITLE
Awesome Books Using ES6

### PR DESCRIPTION
**Pull request by FaranoshAmini**
In this pull request, I have followed these requirements :
     - I have initialized npm
     - I have created index.html, index.css, and index.js
     - I have divided my javascript codes into different separated modules and imported them into the index.js
     - I have used the Luxon library
     - I have used the ES6 syntax

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,59 @@
...
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+You must have a browser and a git version installed on your device, and clone the repo from GitHub and use it.
+
+### Usage
+
+Clone the repositry and open the index.html which is in the folder.
+
```
Comment (zieeco) [inline README.md:29]: - Please, kindly add some instruction on how to clone your repo to have your project up and running in the user's local machine to make your readme more professional.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,59 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
```
Comment (zieeco) [inline README.md:46]: - Please, kindly update this section by adding the link to the issues page of your repo to make it easy for the uses to just click the link from here and it takes them to the issues page.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,59 @@
...
+- Hat tip to anyone whose code was used 🔰
+- Inspiration 💘
+- Coding Partners
+- Microverse program ⚡
+- My standup team 🏹
+- My family's support 🙌
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (zieeco) [inline README.md:59]: - Please, kindly update the MIT.md link as of now, the link is broken to make the readme more professional.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @FaranoshAmini ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: MIT.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [YOUR NAME]
+
+###### Please delete this line and the next one
+###### APP TYPE can be a webpage/website, a web app, a software and so on
+
```
Comment (HAMMAS-SALEEM) [inline MIT.md:5]: - [ ] Please remove these line because these are just instructions :+1:

File: MIT.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [YOUR NAME]
+
```
Comment (HAMMAS-SALEEM) [inline MIT.md:2]: - [ ] Please add the current year in place of 2021 :+1:
- [ ] Please add your name in place of **[YOUR NAME]** :blush:

File: MIT.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+###### Please delete this line and the next one
+###### APP TYPE can be a webpage/website, a web app, a software and so on
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this [APP TYPE] and associated documentation files, to deal in the [APP TYPE] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [APP TYPE] is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the [APP TYPE].
+
+THE [APP TYPE] IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE [APP TYPE] OR THE USE OR OTHER DEALINGS IN THE [APP TYPE].
```
Comment (HAMMAS-SALEEM) [inline MIT.md:10]: - [ ] Please add **WEB APP** or **WEBSITE** in place of **[APP TYPE]** :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]

File: MIT.md (hunk 6 above)
Comment (HAMMAS-SALEEM) [inline MIT.md:10]: - [ ] Please add **WEB APP** or **WEBSITE** in place of **[APP TYPE]** :+1:

Comment (FaranoshAmini) [reply MIT.md]: I already add it.

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @FaranoshAmini, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: MIT.md (hunk 6 above)
Comment (FaranoshAmini) [reply MIT.md]: @HAMMAS-SALEEM  Thanks a lot for your kind review and feedback.

**Pull request merged**